frontend/src/app/agent-edge.css

```
@@ -0,0 +1,376 @@
+     anybody watches. */
+  animation:
+    var(--bodyDrift) var(--bodyDriftTime) ease-in-out infinite,
+    agentedgeswell var(--bodySwellTime) ease-in-out infinite;
+}
+/* Cool, and the slowest: this is the one that sets the base colour of a side for
+   long enough to be noticed. */
+.agentedge-body-1 {
+  --bodyDrift: agentedgedrift1;
+  --bodyDriftTime: 17s;
+  --bodySwellTime: 11s;
+  left: -18%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)